packages/nextjs/app/page.tsx

```diff
@@ -1,67 +1,59 @@
 "use client";
 
-import Link from "next/link";
+import { useRouter } from "next/navigation";
 import type { NextPage } from "next";
-import { useAccount } from "wagmi";
-import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
-import { Address } from "~~/components/scaffold-eth";
+import { PlusCircleIcon } from "@heroicons/react/24/outline";
+import { AuditList } from "~~/components/AuditList";
+import { useHydrateDraftAudits } from "~~/services/store";
 
 const Home: NextPage = () => {
-  const { address: connectedAddress } = useAccount();
+  const router = useRouter();
+  const { draftAudits, addDraftAudit, hydrated } = useHydrateDraftAudits();
+
+  function createNewAudit() {
+    const defaultTitle = "Untitled_Audit";
```

Comment:
Maybe no default title or `Draft created at 02.09.2024` would be slightly more useful than a bunch of "Untitled_Audit" (as soon as the user used it for a while)